notebooks/sparse_grid_ao.ipynb

```diff
@@ -160,7 +160,7 @@
     "            \n",
     "            for line in output.split(\"\\n\"):\n",
```

Comment:
Did we check if the unit [eV] was correct (or if it should be [meV])? I think it'd be neat to have a cut-off around 42meV (chemical accuracy). If the units are [eV] then we're probably sparsifying a bit too much (150eV error is >1000x worse error than ML models).

Reply:
Yes, the unit was correct. The high error we saw was for 32 carbon atoms placed in a line - the error for placing atoms in a line is high regardless of the grid sparsification (c_32 on the graph). 

![image](https://github.com/graphcore-research/pyscf-ipu/assets/102740547/61568e9c-b4f5-42aa-9939-b716e14c4dc9)